db/seeds.rb

```diff
@@ -35,7 +35,12 @@ def create_artists
                    last_name: last_name,
                    username: "#{first_name.downcase}_artist",
                    password: "password",
-                   role: 1)
+                   role: 1,
+                   email_address: "#{first_name.downcase}_artist@gmail.com",
+                   street_address: "123 Maple Drive",
+                   city: "Denver",
+                   state: "CO",
+                   zipcode: 80231)
```

Comment:
I don't think we need to do this with a zipcode, it doesn't make any sense and makes it more confusing to read.